tests/Fixer/ClassNotation/OrderedClassElementsFixerTest.php

```diff
@@ -373,53 +373,6 @@ class Foo
         ];
     }
 
-    /**
-     * @param string $expected
-     *
-     * @group legacy
-     * @dataProvider provideConfigurationCases
-     * @expectedDeprecation Passing "order" at the root of the configuration is deprecated and will not be supported in 3.0, use "order" => array(...) option instead.
```

Comment:
why this removed? I think this might be a BC break like this?

Reply:
yeah
dropping the tests itself is not a bc breaker, but dropping the code that make this test pass is a bc breaker:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3448/files#r161489183

Reply:
I removed that exactly because I replaced `FixerConfigurationResolverRootless` with `FixerConfigurationResolver`. I did that because I noticed that `FixerConfigurationResolverRootless` is deprecated and will be removed in 3.0 and didn't know of any other way to do what I needed.